cluster/gce/gci/configure-helper.sh

```diff
@@ -206,8 +207,10 @@ function get-or-generate-uuid(){
 #Formats the given device ($1) if needed and mounts it at given mount point
 # ($2).
 function safe-format-and-mount() {
-  local device="${1}"
-  local mountpoint="${2}"
+  local device
+  local mountpoint
+  device=$1
+  mountpoint=$2
```

Comment:
Probably want to carry the double quotes forward here

Reply:
Quotes are superfluous for assignments.

```
$ a='foo bar'
$ b=$a
$ echo $b
foo bar
$
```

Reply:
This is a practice defined by the style guide the project's coding conventions reference. Here's the specific section:

https://google.github.io/styleguide/shellguide.html#quoting

Reply:
I read this:

```
# "prefer quoting words", not compulsory
readonly USE_INTEGER='true'

```

But if you want it, you shall have it. Update coming up.